tests/test_examples/test_tutorials_basic.py

```diff
@@ -53,7 +53,8 @@ def test_tutorial_working_directory(tmpdir: Path) -> None:
         "hydra.job.chdir=True",
     ]
     result, _err = run_python_script(cmd)
-    assert result == "Working directory : {}".format(tmpdir)
+    working_directory, output_directory = result.split("\n")
+    assert working_directory == "Working directory : {}".format(tmpdir)
```

Comment:
Might as well modernize it while you are here.
```suggestion
    assert working_directory == f"Working directory : {tmpdir}"
```

Reply:
You are right! I (mistakingly) thought that f-strings is 3.7+ feature :)

Reply:
FYI @shagunsodhani Hydra has basically dropped support for python3.6 at this point.
The `.circleci/config.yml` and `setup.py` files only mention versions 3.7-3.11.

Reply:
Thanks for the clarification. Should we also remove the references to 3.6 from [nox file](https://github.com/facebookresearch/hydra/blob/main/noxfile.py#L18) and plugins ?

Reply:
> Should we also remove the references to 3.6 from [nox file](https://github.com/facebookresearch/hydra/blob/main/noxfile.py#L18) and plugins ?

Sure, I think that would make sense. I'll submit a PR.